### PR TITLE
UHM-8098: add GPAL to diagnosis form

### PR DESCRIPTION
**Pull request by cioan**
@lnball , I got the new diagnosis form to display the GPAL and GA values:

Screenshot 2024-08-02 at 10 17 30 AM

@mseaton , I also switched the mch.js to use class names and I tested a few forms:

Screenshot 2024-08-01 at 3 45 11 PM

@lnball , can you please make sure all forms that contain this code work as expected? Thanks!
Soon, I will start my kgh environment again and verify those forms there too.

### CODE REVIEW

File: configuration/pih/htmlforms/section-dx-obgyn.xml [hunk 1]
```diff
@@ -0,0 +1,307 @@
+<htmlform formName="Diagnostic with obgyn" formVersion="1.0"
```
Comment (mseaton) [inline configuration/pih/htmlforms/section-dx-obgyn.xml:1]: Why is this form being added in this PR?  I thought this was just about adding GPAL calculation?

File: configuration/pih/scripts/visit/encounterTypeConfig.js [hunk 2]
```diff
@@ -111,6 +111,17 @@ angular.module("encounterTypeConfig", [])
             editUrl: "/htmlformentryui/htmlform/editHtmlFormWithStandardUi.page?patientId={{visit.patient.uuid}}&visitId={{visit.uuid}}&encounterId={{encounter.uuid}}&definitionUiResource=" + getFormResource("section-dx.xml") + "&returnUrl={{returnUrl}}&breadcrumbOverride={{breadcrumbOverride}}"
         };
 
+        var obgynDx = {
```
Comment (mseaton) [inline configuration/pih/scripts/visit/encounterTypeConfig.js:114]: Why are these changes part of this PR?

File: configuration/pih/scripts/global/mch.js [hunk 3]
```diff
@@ -26,16 +26,16 @@ function setUpEdd(currentEncounterDate, msgWeeks) {
...
+      jq(".calculated-gestational-age-value").text(gestAgeText);
     } else {
-      jq("#calculated-edd-and-gestational").hide();
+      jq(".calculated-edd-and-gestational").hide();
     }
   };
 
-  jq("#calculated-edd-and-gestational").hide();
+  jq(".calculated-edd-and-gestational").hide();
   jq("#lastPeriodDate input[type='hidden']").change(function () {
```
Comment (mseaton) [inline configuration/pih/scripts/global/mch.js:39]: This all looks good assuming it has been tested to work across all forms in which this feature is used.

File: configuration/pih/htmlforms/section-dx-obgyn.xml (hunk 1 above)
Comment (cioan) [reply configuration/pih/htmlforms/section-dx-obgyn.xml:1]: @mseaton , this form came from this PR  https://github.com/PIH/openmrs-config-pihemr/pull/520/files  that @lnball created.

File: configuration/pih/scripts/global/mch.js (hunk 3 above)
Comment (cioan) [reply configuration/pih/scripts/global/mch.js:39]: Thanks!

**Pull request merged**